src/Microsoft.AspNet.Mvc.Xml/Formatters/XmlOutputFormatter.cs

```diff
@@ -73,5 +76,32 @@ public static XmlWriterSettings GetDefaultXmlWriterSettings()
         {
             return XmlWriter.Create(writeStream, xmlWriterSettings);
         }
+
+        private Type TryGetSerializableErrorType(Type type)
+        {
+            if (type == null)
+            {
+                return null;
+            }
+
+            if (type.Equals(SerializableErrorType))
```

Comment:
my bad..changing to `type ==`...isassignablefrom is not required as SerializableError is sealed...